lib/crypto/base36.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var crypto = require('crypto')
+
+function base36(len) {
+  var out = ''
+  while (out.length < len) {
+    var rand = crypto.randomBytes(len)
```

Comment:
TBH I thought we already had such an issue, but I can't find one.  I filed one at https://github.com/mozilla/fxa-auth-server/issues/1474